File: src/lab.js
const crypto = require('crypto');

function hash(text) {
  return crypto.createHash('sha256').update(text).digest();
};

const results = [];

for (let i = 0; i < 10; i++) {
  const start = Date.now();
  for (let j = 0; j < 1000000; j++) {
    hash('hello world');
  };
  const end = Date.now();

  results.push(end - start);
};

console.log(results.reduce((a, b) => a + b) / results.length);

File: src/lab.py
import hashlib
import functools
import time

def hash(text):
  m = hashlib.sha256()
  m.update(text)
  return m.digest()

results = []

for i in range(10):
  start = time.time() * 1000
  for j in range(1000000):
    hash('hello world'.encode('utf-8'))
  end = time.time() * 1000
  results.append(end - start)

print(functools.reduce(lambda a, b: a + b, results) / 10, end="")